packages/main/src/components/protected/sidebar/styles/Drop.module.css

```diff
@@ -7,18 +7,20 @@
 .item__list:hover,
 .dropDown::selection {
   background-color: #f1fffa;
+  color: black;
 }
 
 .dropdown_active,
 .item__list__active {
   /* width: 90%; */
-  border-radius: 15px;
+  border-radius: 4px;
   background-color: #7ed5af;
 }
 
 .dropdown_active p.dropdown__room,
 .item__list__active div.dropDown__name {
-  color: #000;
+  /* color: #000; */
```

Comment:
If the code is not needed, instead of commenting it out, do away with it entirely